### PR TITLE
Add cluster ID as extra label for newly created PDs.

### PR DESCRIPTION
**Pull request by tsmetana**
Use the new `extra-labels` option introduced in https://github.com/openshift/gcp-pd-csi-driver/pull/11.

### CODE REVIEW

File: assets/controller.yaml [hunk 1]
```diff
@@ -33,6 +33,7 @@ spec:
             - --endpoint=$(CSI_ENDPOINT)
             - --logtostderr
             - --v=${LOG_LEVEL}
+            - --extra-labels=${CLUSTER_ID}=owned
```
Comment (bertinatto) [inline assets/controller.yaml:36]: Do you need to add pass in [extra-create-metadata](https://github.com/kubernetes-csi/external-provisioner/blob/4280d5747a58e8d12430745cc1a42eb8fa0d19f0/cmd/csi-provisioner/csi-provisioner.go#L77) to csi-provisioner as well?

Comment (tsmetana) [reply assets/controller.yaml:36]: No, I don't really care about PV/PVC here. It's the label on PD i need to be created, ie. the disk in GCE not the Kubernetes object.

**Comment by tsmetana** [comment]
/hold
The new CSI driver doesn't seem to be in the CI images yet...

**Pull request merged**